tempesta_fw/procfs.c

```diff
@@ -137,6 +151,145 @@ tfw_perfstat_seq_open(struct inode *inode, struct file *file)
 	return single_open(file, tfw_perfstat_seq_show, PDE_DATA(inode));
 }
 
+/*
+ * Individual server statistics. Note that 1%, 50%, and 99% percentiles
+ * are used to tell the minimum, the median, and the maximum values
+ * correspondingly.
+ */
+static const PrcntlStats __read_mostly tfw_procfs_pstats[] = {
+	{1}, {50}, {75}, {90}, {95}, {99}
```

Comment:
No, the comment is wrong. Minimum and maximum have nothing with percentilies. Consider sequence

        0 100 101 102 103 ... 297 1000      (200 numbers in total)

minimum and maximums are 0 and 1000 correspondingly, while 1st percentile is 100 and 99th is 297 correspondingly  much different numbers.